Content.Server/Chemistry/EntitySystems/SolutionContainerSystem.cs

```diff
@@ -333,6 +335,27 @@ public FixedPoint2 GetReagentQuantity(EntityUid owner, string reagentId)
         return reagentQuantity;
     }
 
+    public bool TryGetMixableSolution(EntityUid uid,
+    [NotNullWhen(true)] out Solution? solution,
+    SolutionContainerManagerComponent? solutionsMgr = null)
```

Comment:
Wrong indentation, check how your IDE wants this indented assuming it's not visual studio. (If it is, ill look for a correct example later)

Reply:
fixed!